crates/autopilot/src/arguments.rs

```diff
@@ -298,6 +303,51 @@ impl std::fmt::Display for Arguments {
     }
 }
 
+#[derive(clap::Parser, Clone)]
+pub struct FeePolicy {
+    /// How much of the order's price improvement over max(limit price,
+    /// best_bid) should be taken as a protocol fee.
+    #[clap(
+        long,
+        env,
+        default_value = None,
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub price_improvement_factor: Option<f64>,
+
+    /// How much of the order's volume should be taken as a protocol fee.
+    #[clap(
+        long,
+        env,
+        default_value = None,
+        value_parser = shared::arguments::parse_percentage_factor
+    )]
+    pub volume_factor: Option<f64>,
+
+    /// Should protocol fees be collected or skipped for limit orders with
+    /// in-market price at the time of order creation.
+    #[clap(long, env, action = clap::ArgAction::Set, default_value = "true")]
+    pub skip_in_market_orders: bool,
+
+    /// Should protocol fees be collected or skipped for TWAP limit orders.
+    #[clap(long, env, action = clap::ArgAction::Set, default_value = "true")]
+    pub skip_twap_orders: bool,
```

Comment:
Can we remove this for now? I'd still argue that since TWAP isn't a concept in the core CoW Protocol domain, we shouldn't define a fee policy for it. Instead, we can consider the TWAP UI an integration on top of CoW Protocol that defines the fee.

Reply:
This parameters does not define if we should use fee policies defined in TWAP order. It defines if we should apply the general limit order fee policies to twap orders.

I can remove it but anyway I need to skip twap orders when forwarding fee policies to the driver.

Reply:
removed.